workloads/logging/deploy_logging_stack.sh

```diff
@@ -28,8 +28,13 @@ fi
 
 function install() {
   # create cluster logging and elasticsearch resources
-  log "Creating cluster logging and elastisearch resources"
-  envsubst < ./files/logging-stack.yml | oc create -f -
+  if [[ $DEPLOY_ES == "true" ]]; then
+    log "Creating cluster logging and elastisearch resources"
+    envsubst < ./files/logging-stack.yml | oc create -f -
+  elif [[ $CUSTOM_ES_URL != "" ]]; then
```

Comment:
What's the use case for using custom ES?

Reply:
If you want to point it to an off-cluster ES server you would need to do this. The simplest example being our CI which points to our dev ES instance for the log test. That way we don't need to eat up space on the cluster.

Reply:
Got it, thanks.